L1Trigger/L1TMuonEndCap/plugins/L1TMuonEndCapTrackProducer.cc

```diff
@@ -41,6 +41,7 @@ using namespace L1TMuon;
 L1TMuonEndCapTrackProducer::L1TMuonEndCapTrackProducer(const PSet& p) {
 
   inputTokenCSC = consumes<CSCCorrelatedLCTDigiCollection>(p.getParameter<edm::InputTag>("CSCInput"));
+  bxShiftCSC = p.getUntrackedParameter<int>("CSCInputBxShift", 0);
```

Comment:
Hi Andrew,
the rule (guideline ?) is that parameters that affect the (physics) results should not be Untracked.
Only parameters that affect things like file names, verbosity level, etc.

The right syntax is just
```c++
bxShiftCSC = p.getParameter<int>("CSCInputBxShift");
```